variants/CATENA_461x/variant.h

```diff
@@ -152,14 +146,24 @@ enum {
 //
 // SERIAL_PORT_HARDWARE_OPEN  Hardware serial ports which are open for use.  Their RX & TX
 //                            pins are NOT connected to anything by default.
-#ifdef USBCON
+#if defined (USBCON) && defined (USB_TWO_HWSERIAL)
+# define SERIAL_PORT_MONITOR  SerialUSB
+# define HAVE_HWSERIAL1
+# define HAVE_HWSERIAL2
+#elif defined (USBCON)
 # define SERIAL_PORT_MONITOR  SerialUSB
 #elif defined(NO_HWSERIAL)
 # define SERIAL_PORT_MONITOR  SerialNo
 #else
```

Comment:
Need a comment on the `#else`.
```suggestion
#else /* ! defined(USBCON) && !defined(NO_HWSERIAL) */
```